boskos/BUILD.bazel

```diff
@@ -13,6 +16,27 @@ go_binary(
     pure = "on",
 )
 
+go_image(
```

Comment:
This will use a distroless image by default, is that okay?

Reply:
yes that s fine.